ert_gui/simulation/ensemble_experiment_panel.py

```diff
@@ -15,6 +15,10 @@
 from ert_gui.simulation.simulation_config_panel import SimulationConfigPanel
 from ert_shared.models import EnsembleExperiment
 
+from collections import namedtuple
+
+Arguments = namedtuple("Arguments", ["mode", "iter_num", "realizations"])
```

Comment:
How about using `@dataclass` instead? Personally I find it easier to read:
```suggestion
@dataclass
class Arguments:
    mode: str
    iter_num: int
    realisations: str
```
?

Reply:
I will give it a go

Reply:
Used @dataclass, I agree it looks more readable